services/udp/service.go

```diff
@@ -137,8 +137,8 @@ func (s *Service) serve() {
 	defer s.wg.Done()
 
 	s.batcher.Start()
+	buf := make([]byte, UDPBufferSize)
```

Comment:
yes, please just revert this part of the change, it'll be changing in my udp-perf branch anyways :+1: